static/app/views/settings/organizationMembers/organizationMembersList.tsx

```diff
@@ -195,6 +292,35 @@ class OrganizationMembersList extends AsyncView<Props, State> {
             })
           }
         </ClassNames>
+        {inviteRequests && inviteRequests.length > 0 && (
+          <Panel>
+            <PanelHeader>
+              <StyledPanelItem>
+                <div>{t('Pending Members')}</div>
+                <div>{t('Role')}</div>
+                <div>{t('Teams')}</div>
+              </StyledPanelItem>
+            </PanelHeader>
+            <PanelBody>
+              {inviteRequests.map(inviteRequest => (
+                <InviteRequestRow
+                  key={inviteRequest.id}
+                  organization={organization}
+                  inviteRequest={inviteRequest}
+                  inviteRequestBusy={{}}
+                  allTeams={teams}
+                  allRoles={currentMember?.roles ?? MEMBER_ROLES}
+                  onApprove={this.handleInviteRequestApprove}
+                  onDeny={this.handleInviteRequestDeny}
+                  onUpdate={data => this.updateInviteRequest(inviteRequest.id, data)}
+                />
+              ))}
+              {inviteRequests.length === 0 && (
```

Comment:
@Neo-Zhixing I don't think yo need this condition anymore; we just won't render anything.

Reply:
fixed